src/comm_manager.cpp

```diff
@@ -281,6 +282,19 @@ void CommManager::attitude_correction_callback(const turbomath::Quaternion &q)
 {
   RF_.estimator_.set_attitude_correction(q);
 }
+void CommManager::heartbeat_callback(void)
+{
+  static bool error_data_sent = false;
```

Comment:
If I understand correctly, the idea behind this logic is that every time the flight controller starts up, when it receives its first heartbeat message from `rosflight_io` it will send backup data if it has it. This is the behavior we want immediately following a hard fault, but it seems like once something ever gets written to backup SRAM, this logic will always send that crash report after reboot, even if the system has been legitimately turned off and back on again, even multiple times. So do we need to delete or invalidate the contents of the backup SRAM after we send it to avoid that behavior?